docs/reference/query-dsl/function-score-query.asciidoc

```diff
@@ -321,14 +325,17 @@ There are a number of options for the `field_value_factor` function:
     Value used if the document doesn't have that field. The modifier
     and factor are still applied to it as though it were read from the document.
 
+IMPORTANT: Scores produced by the `field_value_score` function must be
```

Comment:
This is just personal preference (feel free to ignore), but this got me thinking when we use `IMPORTANT:` vs. `NOTE:`. This and the following paragraph seem more like notes to me as it's a pretty routine description of a limitation.

Reply:
Good call, I've changed these to `NOTE`.